dspace/config/submission-forms_cs.xml

```diff
@@ -85,7 +85,7 @@
                     <dc-qualifier>author</dc-qualifier>
                     <repeatable>true</repeatable>
                     <label>Author</label>
```

Comment:
???

Reply:
The whole submission-form_cs.xml will be synchronized, but not in this PR.